server/src/lib/models/JWTDenyList.ts

```diff
@@ -0,0 +1,39 @@
+import {
+	Model,
+	Column,
+	Table,
+	IsUUID,
+	PrimaryKey,
+	DataType,
+	CreatedAt,
+	UpdatedAt,
+	DeletedAt,
+	Unique
+} from "sequelize-typescript"
+
+@Table
+export class JWTDenyList extends Model {
+	@IsUUID(4)
+	@PrimaryKey
+	@Unique
+	@Column({
+		type: DataType.UUID,
+		defaultValue: DataType.UUIDV4
+	})
+	id!: string
+
+	@Column
+	token!: string
+
+	@CreatedAt
+	@Column
+	createdAt!: Date
+
+	@UpdatedAt
+	@Column
+	updatedAt!: Date
+
+	@DeletedAt
+	@Column
+	deleteddAt!: Date
```

Comment:
can we add a `reason` column? It can be a required string. Eventually we may want to know if it was manually revoked or expired.